fog/ledger/server/tests/router_connection.rs

```diff
@@ -863,6 +863,281 @@ fn fog_ledger_untrusted_tx_out_api_test(logger: Logger) {
     sleep(Duration::from_millis(1000));
 }
 
+// Test that a fog ledger connection is able to check key images by hitting
+// a fog ledger router using the unary API
+#[test_with_logger]
+fn fog_router_unary_key_image_test(logger: Logger) {
```

Comment:
We'll need to remove the test set up logic from this test because it makes it hard to read. Can you make an issue to track this work?

Reply:
Also #3168